consensus/types_internal.go

```diff
@@ -18,6 +18,10 @@ func (bh *blockHash) GetValue() [32]byte {
 	return bh.value
 }
 
+func (bh *blockHash) Eq(other BlockHash) bool {
+	return bh.value == other.GetValue()
```

Comment:
Why not bytes.Equal?

Reply:
Ended up deleting this function entirely since it's not used. But you're right that bytes.Equal(blockHash[:]) is the right solution. I'll use that later in the stack